templates/solutions/ai/index.html

```diff
@@ -0,0 +1,571 @@
+      <hr />
+      <p>
+        <a href="https://ubuntu.com/engage/mlops-toolkit">Download our MLOps toolkit&nbsp;&rsaquo;</a>
+      </p>
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/06e389f7-Image-1.png",
+        alt="",
+        width="1200",
+        height="676",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Run on any cloud</h2>
+    </div>
+    <div class="col">
+      <p>Run your workloads anywhere, including hybrid and multi-cloud environments.</p>
+      <p>Choose the ideal infrastructure for your use cases &mdash; start quickly with no risk and low investment on public cloud, then move workloads to your own data centre as you scale. Benefit from a consistent, optimised OS, tooling and GPU support throughout your entire AI journey.</p>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <div class="p-logo-section u-realign-logos u-align--right">
+      <div class="p-logo-section__items">
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/7c4a4785-microsoft-azure-logo.png",
+            alt="Microsoft Azure",
+            width="335",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/145d9c5d-ibm-cloud-logo.png",
+            alt="IBM Cloud",
+            width="538",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/a5b8bc4f-aws-logo.png",
+            alt="AWS",
+            width="187",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/55ef5a98-google-cloud-logo.png",
+            alt="",
+            width="595",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item" style="margin-right: 0;">
```

Comment:
Using `u-no-margin--right` would be cleaner.

```suggestion
        <div class="p-logo-section__item u-no-margin--right">
```